ground/gcs/src/app/main.cpp

```diff
@@ -264,6 +265,9 @@ int main(int argc, char **argv)
         // https://bugreports.qt-project.org/browse/QTBUG-32789
         QFont::insertSubstitution(".Lucida Grande UI", "Lucida Grande");
     }
+    QProcess process;
```

Comment:
Excellent sleuthing! Could your findings be documented in the code and in the git commit message? I fear it will otherwise turn into an unmaintainable piece of code.